contracts/src/v0.8/dev/keeper2_0/KeeperRegistry2_0.sol

```diff
@@ -425,28 +433,41 @@ contract KeeperRegistry2_0 is
     override
     returns (
       State memory state,
-      Config memory config,
-      address[] memory keepers
+      OnChainConfig memory config,
```

Comment:
nit: In OCR we standardized to use "Onchain" not "OnChain"

Reply:
fixed